projects/plugins/protect/src/js/components/stat-card/styles.module.scss

```diff
@@ -0,0 +1,59 @@
+.wrapper {
+	background-color: var( --jp-white );
+	box-shadow: 0px 4px 40px rgba(0, 0, 0, 0.08);
+	border-radius: calc( var( --jp-border-radius ) * 2 ); // 8px;
+	display: flex;
+
+	&.horizontal {
+		align-items: center;
+		padding: calc( var( --spacing-base ) * 2 ); // 16px;
+		width: 358px;
+
+		.info {
+			display: flex;
+			align-items: center;
+			width: 100%;
+			overflow: hidden;
+		}
+
+		.label {
+			margin: 0 calc( var( --spacing-base ) * 2	); // 16px;
+			flex-grow: 1;
+		}
+
+		.icon {
+			margin-bottom: 0;
+		}
+	}
+	
+	&.square {
+		--square-stat-vertical-spacing: calc( var( --spacing-base ) * 2	); // 16px;
+		--square-stat-horizontal-spacing: calc( var( --spacing-base ) * 3	); // 24px;
+		flex-direction: column;
+		justify-content: space-between;
+		width: fit-content;
+		padding: var( --square-stat-vertical-spacing ) var( --square-stat-horizontal-spacing );
+
+		.label {
+			white-space: nowrap;
+		}
+	}
+
+	&.disabled {
+		opacity: .5;
+	}
+
+	.value {
+		overflow: hidden;
+		text-overflow: ellipsis;
```

Comment:
I think this is okay for now, but when we integrate this component into the header, we should make sure it can support displaying numbers up to a certain expected amount of digits without needing the ellipsis. :+1:

Reply:
Ah, I see it uses shorthand numbers like "10K" instead of "10,000", all good :+1: :+1: